### ISSUE TITLE
Fix to disable LoadFromFile with folder path

### ISSUE THREAD

**Issue by Alexey-T** [BerndGabriel/HtmlViewer#290]
if I browse local html files, load local html file to Frame, and click link, LoadFromFile called with folder name (it's relative dir in A tag).
This crashes in procedure TBuffer.SetStream(Stream: TStream);   
"range check error" on reading filename.

Linux x64, Lazarus 2.1

Fix: framview.pas
```
procedure TFrameSet.LoadFromFile(const FName, Dest: ThtString);
var
  I: Integer;
  Frame: TfvFrame;
  Lower, Upper: Boolean;
  EV: EventRec;
  EventPointer: PEventRec;
  Stream: TStream;
  ft: ThtmlFileType;
begin
  Clear;
  NestLevel := 0;
  if not FileExists(FName) then Exit;
  if DirectoryExists(FName) then Exit;
  ft := HTMLType;                  
```

**Comment by Alexey-T** [comment]
DirectoryExists needed in Laz, because FPC 3.0.4 gets FileExists(s)=True for folder.

**Comment by BerndGabriel** [comment]
Fixed in Version 11.10
